server/src/main/java/org/elasticsearch/index/mapper/SourceFieldMapper.java

```diff
@@ -468,7 +412,7 @@ protected String contentType() {
 
     @Override
     public FieldMapper.Builder getMergeBuilder() {
-        return new Builder(indexMode, Settings.EMPTY, false).init(this);
+        return new Builder(null, Settings.EMPTY, false).init(this);
```

Comment:
Null is provided here as indexMode. It looks weird, but I don't think it is harmful. Since none of the attributes are really updatable.

Reply:
Yes, and `init` that is called afterwards "resets" all parameters anyway.